### PR TITLE
Adds the ability to specify the appserver host address by an environment variable.

### PR DESCRIPTION
**Pull request by wilford**
Checks the DEV_APPSERVER_DEFAULT_ADDRESS environment variable for a default appserver address.

### CODE REVIEW

File: src/main/groovy/com/google/appengine/AppEnginePluginExtension.groovy [hunk 1]
```diff
@@ -26,7 +26,10 @@ import org.gradle.api.Project
  * @author Benjamin Muschko
  */
 class AppEnginePluginExtension {
-    String httpAddress = "localhost"
+    String httpAddress =
```
Comment (loosebazooka) [inline src/main/groovy/com/google/appengine/AppEnginePluginExtension.groovy]: Can we use the elvis operator here from groovy?
http://docs.groovy-lang.org/latest/html/documentation/index.html#_elvis_operator

```
String httpAddress = System.getenv("DEV_APPSERVER_DEFAULT_ADDRESS") ?: "localhost"
```

Also not using standard google line lengths in this file, so feel free to go up to 120.

Comment (wilford) [inline src/main/groovy/com/google/appengine/AppEnginePluginExtension.groovy]: Done.

**Comment by wilford** [comment]
Addressed comments.

**Comment by loosebazooka** [comment]
Cool, I'll merge it when I'm back in the office

**Comment by loosebazooka** [comment]
this will go out in the next release.

**Pull request merged**